src/frontend/apps/lti_site/components/VideoPlayer/index.tsx

```diff
@@ -60,7 +65,6 @@ const VideoPlayer = ({
    * Noop out if the video is missing, render will redirect to an error page.
    */
   useEffect(() => {
```

Comment:
React Hook useEffect has missing dependencies: 'defaultVolume', 'intl.locale', 'languages', 'playerType', 'setPlayerCurrentTime', 'timedTextTracks', and 'video'. Either include them or remove the dependency array  react-hooks/exhaustive-deps

Lot of dependencies are missing in this useEffect, it is maybe the good time to have a look at it. What do you think @lunika?

Reply:
Here as well by the way:
- Line 105 - warning  React Hook useEffect has a missing dependency: 'video.urls.manifests'. Either include it or remove the dependency array.
- Line 138 - warning  React Hook useEffect has a missing dependency: 'languages'. Either include it or remove the dependency array

Reply:
The commande to see the errors:
`cd src/frontend/apps/standalone_site && yarn eslint ../lti_site/components/VideoPlayer/index.tsx -c .eslintrc.js  --parser-options "project: ['../../apps/lti_site/tsconfig.json']"`

Reply:
No, just this file; it is not Alfred who made these useEffect, but if he is doing a refactoring of this file, it is maybe a good moment to have a look at them.